docker/everest-playground-ubuntu-20-04/Dockerfile

```diff
@@ -0,0 +1,72 @@
+ARG BASE_IMAGE=ubuntu
+ARG BASE_IMAGE_VERSION=20.04
+FROM ${BASE_IMAGE}:${BASE_IMAGE_VERSION}
+
+ARG USERNAME=docker
+ARG USER_UID=1000
+
+# configure timezone data
+ENV TIMEZONE=Europe/Berlin
+RUN ln -snf /usr/share/zoneinfo/$TIMEZONE /etc/localtime && echo $TIMEZONE > /etc/timezone
+
+# create user and add to sudoers
+RUN useradd -ms /bin/bash -u ${USER_UID} -U ${USERNAME}
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y sudo \
+    && echo ${USERNAME} ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/${USERNAME} \
+    && chmod 0440 /etc/sudoers.d/${USERNAME} \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y \
+    git \
+    rsync \
+    wget \
+    cmake \
+    doxygen \
+    graphviz \
+    build-essential \
+    clang-tidy \
+    cppcheck \
+    libboost-all-dev \
+    maven \
+    openjdk-11-jdk \
+    libsqlite3-dev \
+    python3-pip \
+    libssl-dev \
+    gpg \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
The codacy comment concerning pinning the package numbers is indeed helpful.  It might be quiet some work but would lead to a more "reproducable" container

Reply:
Sounds like a good idea to me, I'll have a look